react-context/flow.txt

```diff
@@ -0,0 +1,19 @@
+User can sign up:
+The user interacts with the client and inputs a unique username with a password. They are sent to the server and if they're viable,
+the password is hashed by argon2. Then, the username and hashed password is queried as a `select` request to be added to the database.
+The database sends a response back to the server, which then informs the client whether the username and password were
+successfully updated to the database or not.
+
+User can sign in:
+The user interacts with the client and inputs a unique username with a password. If they're viable, they are sent to the server.
+In the server, it queries a `select` request to the database and grabs the userId and hashedPassword using the client's username if it is in there.
+The server then verifies whether or not the hashedPassword is equal to the client's password. If it's matching,
+the user is given a special token with the userId, their username, and the TOKEN_SECRET.
```

Comment:
These parts are pretty good, but please include what the client does when it receives a response from the server.